### ISSUE TITLE
About human pose model

### ISSUE THREAD

**Issue by miaochenGuo** [openvinotoolkit/training_extensions#171]
It's so weird that using openvino to infer, it can get 20fps; but when i use pytorch model .pth, only 1.2fps can i get.

**Comment by Daniil-Osokin** [comment]
Because OpenVINO rocks :fire:! Actually there can be two issues:
1. PyTorch built in not optimized for cpu inference way. Also OpenVINO does some optimizations, which reduces network complexity (e.g. batch normalization layers fusion), this may cause difference in inference time too.
2. Demo has post-processing of network outputs, which may run slowly in Python (but fast in C++), this usually happens if number of persons in frame is big. So low fps may be due to post-processing, not the network inference.

**Comment by Daniil-Osokin** [comment]
@Ilya-Krylov Thanks for the housekeeping! :wink:

**Comment by druzhkov-paul** [comment]
![image](https://user-images.githubusercontent.com/6965254/110685018-e474ee00-81ee-11eb-94c4-384bae81c9b0.png)
